src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -1060,18 +1064,7 @@ public RepairUnit getRepairUnit(UUID id) {
     stmt.setIdempotent(Boolean.TRUE);
     ResultSet results = session.execute(stmt);
     for (Row repairUnitRow : results) {
-      if (repairUnitRow.getString("cluster_name").equals(params.clusterName)
-          && repairUnitRow.getString("keyspace_name").equals(params.keyspaceName)
-          && repairUnitRow.getSet("column_families", String.class).equals(params.columnFamilies)
-          && repairUnitRow.getBool("incremental_repair") == params.incrementalRepair
-          && repairUnitRow.getSet("nodes", String.class).equals(params.nodes)
-          && repairUnitRow.getSet("datacenters", String.class).equals(params.datacenters)
-          && repairUnitRow
-              .getSet("blacklisted_tables", String.class)
-              .equals(params.blacklistedTables)
-          && repairUnitRow.getInt("repair_thread_count") == params.repairThreadCount) {
-
-        repairUnit = RepairUnit.builder()
```

Comment:
why is this condition removed?

Reply:
It is replaced with this check: https://github.com/thelastpickle/cassandra-reaper/pull/1107/files/aca650eef09ee895debd2c87f4e8629e5c47c17f#diff-1eb3c242ea87cb7ce434d35444bd88e1531fe3ad0aae4a3a0e6d96f3411ce56dR1078

The problem here is that we had two different ways of checking if repair units were "equal": the removed code (which is specific to the Cassandra backend) and the `equals()` method in `RepairUnit`.
Now we're left with a single method to check repair unit equality.

Reply:
for permanence: https://github.com/thelastpickle/cassandra-reaper/commit/aca650eef09ee895debd2c87f4e8629e5c47c17f#diff-1eb3c242ea87cb7ce434d35444bd88e1531fe3ad0aae4a3a0e6d96f3411ce56dR1078